Ensure config.h is included first for files that depend on
_FILE_OFFSET_BITS so that no later redefinitions occur which cause compiler
warnings and potentially incorrect program behaviour.

XXX file upstream bug

--- gupnp-0.20.14/libgupnp/gupnp-service.c	2015-05-10 07:38:52.000000000 -0700
+++ gupnp-0.20.14/libgupnp/gupnp-service.c	2015-07-05 21:02:13.767428372 -0700
@@ -27,6 +27,10 @@
  * notification. #GUPnPService implements the #GUPnPServiceInfo interface.
  */
 
+#ifdef HAVE_CONFIG_H
+#   include <config.h>
+#endif
+
 #include <gobject/gvaluecollector.h>
 #include <gmodule.h>
 #include <libsoup/soup-date.h>
--- gupnp-0.20.14/libgupnp/guul.c	2015-04-06 04:45:43.000000000 -0700
+++ gupnp-0.20.14/libgupnp/guul.c	2015-07-05 21:05:13.595134623 -0700
@@ -17,6 +17,10 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#ifdef HAVE_CONFIG_H
+#   include <config.h>
+#endif
+
 #include <stdlib.h>
 
 #include <guul.h>
